_data/simple-icons.json

```
@@ -4525,6 +4525,12 @@
             "source": "https://www.juniper.net/us/en/company/press-center/images/image-library/logos/",
             "guidelines": "https://www.juniper.net/us/en/company/press-center/images/image-library/logos/"
         },
+        {
+            "title": "JUnit5",
+            "hex": "25A162",
+            "source": "https://github.com/junit-team/junit5/tree/86465f4f491219ad0c0cf9c64eddca7b0edeb86f/assets/img",
```

Could you update the `source` entry to the complete URL of the SVG?
```suggestion
            "source": "https://raw.githubusercontent.com/junit-team/junit5/86465f4f491219ad0c0cf9c64eddca7b0edeb86f/assets/img/junit5-logo.svg",
```